src/eko/kernels/QEDnon_singlet.py

```diff
@@ -0,0 +1,364 @@
+# -*- coding: utf-8 -*-
```

Comment:
first trivial comment: rename this file to `non_singlet_qed.py` and similar for the other files here (`_qed` to the end)